app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> None:
+
+    masks_to_buy = 0
+
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The function currently returns immediately if any friend is not vaccinated. This prevents checking the rest of the friends. Consider accumulating the results and returning a summary message after all friends have been checked.